main/options/options.cc

```diff
@@ -481,21 +481,34 @@ void Options::flushPrinters() {
     }
 }
 
-void addFilesFromDir(Options &opts, string_view dir) {
+void addFile(Options &opts, UnorderedSet<string> &knownFileNames, string file) {
```

Comment:
```suggestion
void addFile(Options &opts, UnorderedSet<string> &knownFileNames, const string &file) {
```

Reply:
can the `file` argument be a `string_view`? We prefer them as 'non-owned` strings